source/reference/minio-mc-admin/mc-admin-config.rst

```diff
@@ -1996,26 +1997,14 @@ configuration settings.
    .. mc-conf:: server_addr
       :delimiter: " "
 
-   *Required*
-
-   .. include:: /includes/common-minio-external-auth.rst
-      :start-after: start-minio-ad-ldap-server-addr
-      :end-before: end-minio-ad-ldap-server-addr
-
-   This environment configuration setting with the 
-   :envvar:`MINIO_IDENTITY_LDAP_SERVER_ADDR` environment variable.
-
-   .. mc-conf:: sts_expiry
-      :delimiter: " "
-
-      *Optional*
+      *Required*
 
       .. include:: /includes/common-minio-external-auth.rst
-         :start-after: start-minio-ad-ldap-sts-expiry
-         :end-before: end-minio-ad-ldap-sts-expiry
+         :start-after: start-minio-ad-ldap-server-addr
+         :end-before: end-minio-ad-ldap-server-addr
 
       This environment configuration setting with the 
-      :envvar:`MINIO_IDENTITY_LDAP_STS_EXPIRY` environment variable.
+      :envvar:`MINIO_IDENTITY_LDAP_SERVER_ADDR` environment variable.
```

Comment:
This isn't a complete sentence.
It's all over a lot of the existing page, though. But it's missing something.

Is it supposed to be something like:
> Alternatively, configure this option with the .... environment variable.

Reply:
Oh, wait maybe:

> This configuration option is the same as setting the ...

(And maybe noting that env var override config settings?)

Reply:
There's a "corresponds with" missing in here :grimacing: